pkg/ccl/spanconfigccl/spanconfigsqltranslatorccl/testdata/tenant/full_translate

```diff
@@ -63,59 +63,59 @@ full-translate
-/Tenant/10/Table/1{2-3}                    database system (tenant)
-/Tenant/10/Table/1{3-4}                    database system (tenant)
-/Tenant/10/Table/1{4-5}                    database system (tenant)
-/Tenant/10/Table/1{5-6}                    database system (tenant)
-/Tenant/10/Table/{19-20}                   database system (tenant)
-/Tenant/10/Table/2{0-1}                    database system (tenant)
-/Tenant/10/Table/2{1-2}                    database system (tenant)
-/Tenant/10/Table/2{3-4}                    database system (tenant)
-/Tenant/10/Table/2{4-5}                    database system (tenant)
-/Tenant/10/Table/2{5-6}                    database system (tenant)
-/Tenant/10/Table/2{6-7}                    database system (tenant)
-/Tenant/10/Table/2{7-8}                    database system (tenant)
-/Tenant/10/Table/2{8-9}                    database system (tenant)
-/Tenant/10/NamespaceTable/{30-Max}         database system (tenant)
-/Tenant/10/{NamespaceTable/Max-Table/32}   database system (tenant)
-/Tenant/10/Table/3{2-3}                    database system (tenant)
-/Tenant/10/Table/3{3-4}                    database system (tenant)
-/Tenant/10/Table/3{4-5}                    database system (tenant)
-/Tenant/10/Table/3{5-6}                    database system (tenant)
-/Tenant/10/Table/3{6-7}                    database system (tenant)
-/Tenant/10/Table/3{7-8}                    database system (tenant)
-/Tenant/10/Table/{39-40}                   database system (tenant)
-/Tenant/10/Table/4{0-1}                    database system (tenant)
-/Tenant/10/Table/4{1-2}                    database system (tenant)
-/Tenant/10/Table/4{2-3}                    database system (tenant)
-/Tenant/10/Table/4{3-4}                    database system (tenant)
-/Tenant/10/Table/4{4-5}                    database system (tenant)
-/Tenant/10/Table/4{6-7}                    database system (tenant)
-/Tenant/10/Table/4{8-9}                    database system (tenant)
-/Tenant/10/Table/5{0-1}                    database system (tenant)
-/Tenant/10/Table/5{1-2}                    database system (tenant)
-/Tenant/10/Table/5{2-3}                    database system (tenant)
-/Tenant/10/Table/5{3-4}                    database system (tenant)
-/Tenant/10/Table/5{4-5}                    database system (tenant)
-/Tenant/10/Table/5{5-6}                    database system (tenant)
-/Tenant/10/Table/5{6-7}                    database system (tenant)
-/Tenant/10/Table/5{7-8}                    database system (tenant)
-/Tenant/10/Table/5{8-9}                    database system (tenant)
-/Tenant/10/Table/{59-60}                   database system (tenant)
-/Tenant/10/Table/11{0-1}                   range default
-/Tenant/10/Table/11{1-2}                   range default
-/Tenant/10/Table/11{2-3}                   range default
+[/Tenant/10, /Tenant/10/Table/4)                       database system (tenant)
```

Comment:
It looks like the output is incorrect for this since there are multiple tables being checked.